plugins/github-actions/universal.js

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// This file is shared between frontend and backend plugins
+module.exports.GITHUB_ACTIONS_ANNOTATION = 'github.com/project-slug';
```

Comment:
A thought, could this be a `.mjs` file so that we can use ES6 module syntax?

Reply:
Not super fan of the whole `.mjs` approach, but I think long term we can look into switching towards ES6 modules 100% - https://2ality.com/2019/10/hybrid-npm-packages.html#variant%3A-avoiding-.mjs

Reply:
We can use a package with `type: "module"` as well. Tbh I think it's probably best to move this to a folder either way, similar to https://github.com/spotify/backstage/blob/master/packages/cli/asset-types/package.json.

Reply:
Neither `type:module` or `.mjs` won't work for now, unless we touch a lot of code, e.g. you can't `require` esm code from cjs - https://nodejs.org/api/esm.html#esm_require

But yea, moved it to a separate folder and added package.json for the future updates

Reply:
think this should be:
```js
exports.GITHUB_ACTIONS_ANNOTATION
```

to make rollup happy?